scripts/README.md

```diff
@@ -0,0 +1,177 @@
+help:
+
+```
+pipenv run update --help
+```
+
+
+## Update operations
+
+Note that in order to get any metadata change appear on the frontend you will
+need to [update register metadata on
+registers-frontend](https://github.com/openregister/registers-frontend#updating-register-metadata).
+
+
+### Add data to a register
+
+The update CLI expects a TSV or a CSV, with a header row where each value is
+the exact field name (i.e. identifier) and the rest of rows the the data to
+upload.
+
+```sh
+pipenv update data --rsf=../registry-data/rsf/country.rsf \
+  --phase=test ./new-data.tsv
+```
+
+
+### Change the register title
+
+The name of a register is immutable because it is its unique ID. But the
+register can also have a human-friendly text, the title, which can be updated
+as follows:
+
+```sh
+pipenv run update context --rsf=../registry-data/rsf/country.rsf \
+  --phase=test title "A list of countries recognised by the UK"
+```
+
+Check the context subcommand help for more details:
+
+```sh
+pipenv run update context --help
+```
+
+
+### Change the register description
+
+```sh
+pipenv run update context --rsf=../registry-data/rsf/country.rsf \
+  --phase=test description "A list of countries recognised by the UK"
+```
+
+Check the context subcommand help for more details:
+
+```sh
+pipenv run update context --help
+```
+
+
+### Change the custodian name
+
+```sh
+pipenv run update context --rsf=../registry-data/rsf/country.rsf \
+  --phase=test custodian "Jane Doe"
+```
+
+Check the context subcommand help for more details:
+
+```sh
+pipenv run update context --help
+```
+
+
+### Change a field description
+
+```sh
+pipenv run update schema --rsf=../registry-data/rsf/country.rsf \
+  --phase=test start-date "The date the country was constituted"
+```
+
+Check the context subcommand help for more details:
+
+```sh
+pipenv run update schema --help
+```
+
+
+## Initial load
+
+Once you have an empty ORJ register, you can do an initial load using an RSF
+file.
+
+
+Assuming there is an empty register in the test environment for the
+`web-colour` register you can upload the register by:
+
+
+```sh
+pipenv run update init --phase=test ./web-colour.rsf
+```
+
+
+## The Field and Register registers
+
+
+Instances of ORJ serving the Field and Register Registers read yaml files
+containing fields and registers from S3 on startup.
+
+There is an Ansible task in the deployment repo to generate the files in S3
+from the application. You will need to run this after reloading the metadata
+RSF.
+
+
+## Development
+
+The update CLI is in the `update` python package in this directory. It is
+defined as a Pipenv script in `Pipenv`.
+
+The logic around serialising/deserialising RSF and handling patches and
+registers is in the `registers` python package part of the Registers CLI (See
+https://github.com/openregister/registers-cli).
+
+
+If you need to change how the update CLI handles pass, orj or the CLI itself,
```

Comment:
I would put at least `pass` in backticks so as not to confuse the language meaning and the application name. Also should be ORJ in capitals or write out the whole repo name, to reduce potential confusion.